src/main.cpp

```diff
@@ -48,31 +49,40 @@ void loadFromConfigFile(Config &config)
 {
     std::ifstream inFile("config.txt");
     std::string line;
-    int option;
-    while (inFile >> line >> option) {
+    auto loadInt = [&inFile](int& value) {
+        int option;
+        inFile >> option;
+        value = option;
+    };
+    while (inFile >> line) {
         if (line == "FULLSCREEN") {
-            config.clientOptions.fullScreen = option;
+            loadInt((int&)config.clientOptions.fullScreen);

```

Comment:
Is it ok if we can do this the same as the string thing below as in

```
bool fullscreen;
inFile >> fullscreen;
config.clientOptions.fullScreen = fullscreen;
```

Reply:
I'm using Kiroma's suggested method which just streams the data right into the variable since it follows DRY much better, e.g.:
`inFile > config.clientOptions.fullScreen;`

Reply:
Yeah that's much better 😛